code/game/objects/machinery/vending/quick_vendor.dm

```diff
@@ -130,6 +160,19 @@ GLOBAL_LIST_INIT(quick_loadouts, init_quick_loadouts())
 		return
 	. += emissive_appearance(icon, "[icon_state]_emissive")
 
+/obj/machinery/quick_vendor/beginner //Loadout vendor that shits out basic pre-made loadouts so new players can get something usable
+	icon_state = "Loadoutvendor"
+	categories = list(
+		"Squad Marine",
+		"Squad Engineer",
+		"Squad Corpsman",
+		"Squad Smartgunner",
+	)
+
+/obj/machinery/quick_vendor/beginner/Initialize(mapload)
+	. = ..()
+	global_list_to_use = GLOB.beginner_loadouts
```

Comment:
this technically works, but its far better practice to just have a separate proc that handles any setup, that init calls. That way you can safely just override it without needing to cal parent.
Having init just overriding init is kinda wew.